### DISCUSSION TITLE
Microsoft Edge

### DISCUSSION THREAD

**Discussion by l274** [Prodigy-Hacking/ProdigyMathGameHacking#471] [(A) Anything]
Can You Please Show How To Do It On Microsoft Edge? I Don't have google chrome.

**Comment by PatheticMustan** [comment]
Ah, unfortunately we don't support edge... we're having trouble with even just gchrome 😓

**Reply by ghost** [reply]
I could try making one for Edge.

**Reply by ghost** [reply]
Edge and Opera are chromium-based, it appears. https://docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/

**Comment by l274** [comment]
Ok

**Comment by l274** [comment]
Thanks, I'll check it out!

**Reply by ghost** [reply]
There isn't an Edge extension yet, but I'm working on making one. See #592.

**Comment by l274** [comment]
Awesome, thanks!
